pkg/utils/uuid_test.go

```diff
@@ -0,0 +1,37 @@
+package utils
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewUUIDStr(t *testing.T) {
+	assert := assert.New(t)
+
+	output := NewUUIDStr()
+	outputType := reflect.TypeOf(output)
+	stringType := reflect.TypeOf("")
+
+	assert.IsType(stringType, outputType)
+}
+
+func TestIsValidUUID(t *testing.T) {
+	assert := assert.New(t)
+
+	listUUID := []string{
+		uuid.New().String(),
+		uuid.New().String() + "!xyz"}
+
+	for _, uuid := range listUUID {
+		result := IsValidUUID(uuid)
+
+		if result == false {
```

Comment:
This check isn't needed since we already know result is false in this block. I would recommend using the table driven style where you create an object that maps the input (the string) and the expected output as a boolean